src/mui-setup.js

```diff
@@ -1,7 +1,7 @@
 import { createTheme } from '@mui/material/styles';
 import * as muiConfig from './mui-config.json';
 
-export default function muiSetup(direction) {
+export default function muiSetup(direction, theme) {
```

Comment:
I don't know about this. We don't want this to be dynamic

Reply:
No, we don't. Just found that it is convenient to write like this. But gonna write those into Component.

Reply:
Updated